pages/login.php

```diff
@@ -118,7 +118,11 @@
 
 			<div class="form-elements">
 				<input type="text" name="uid" placeholder="Username/Email">
-				<input type="password" name="pswd" placeholder="Password">
+				<input type="password" name="pswd" id="pswd" placeholder="Password">
```

Comment:
Generally, I avoid using CSS tags inside the HTML element.

Can you add this CSS inside the `<script>` used at the top of this page?

Reply:
done